### PR TITLE
Give source clone pvc a gid so it gets copied properly.

### PR DESCRIPTION
**Pull request by awels**
Signed-off-by: Alexander Wels <awels@redhat.com>

**What this PR does / why we need it**:
Our cloning source pvc file didn't have a gid, so it wouldn't get copied. This PR gives the source file a GID if it applies, so it gets copied and the check properly identifies the gid.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: tests/utils/pod.go [hunk 1]
```diff
@@ -67,6 +67,7 @@ func DeletePod(clientSet *kubernetes.Clientset, pod *k8sv1.Pod, namespace string
 
 // NewPodWithPVC creates a new pod that mounts the given PVC
 func NewPodWithPVC(podName, cmd string, pvc *k8sv1.PersistentVolumeClaim) *k8sv1.Pod {
+	fsGroup := int64(107)
```
Comment (brybacki) [inline tests/utils/pod.go:70]: what exactly is 107?

**Comment by arnongilboa** [comment]
Looks good to me, except the const mentioned by @brybacki .

File: tests/utils/pod.go (hunk 1 above)
Comment (awels) [reply tests/utils/pod.go]: Good point, should have used the constant we have defined for it.

**Pull request merged**